src/datumaro/util/mask_tools.py

```diff
@@ -221,7 +220,12 @@ def lazy_mask(path, inverse_colormap=None):
 
 
 def mask_to_rle(binary_mask):
-    return encode(binary_mask)
```

Comment:
I am concerned about unintentional consequences. If for some reason, we accidentally fail to load the _capi module and execute the encode() function, we may run a slower implementation without realising it. So I think that we should revert this change.

Instead we can create another PR to migrate to use Rust instead of C for the implementation.